src/utils/index.js

```diff
@@ -34,6 +34,18 @@ export const TOO_MANY_FILES_REJECTION = {
   code: TOO_MANY_FILES,
   message: 'Too many files'
 }
+export const FILE_TOO_SMALL_REJECTION = {
```

Comment:
https://github.com/react-dropzone/react-dropzone/issues/988
I have added these exports base on this suggestion 
"In addition, it would be nice if ALL errors were (re-) exported from the utils. Currently, only the TOO_MANY_FILES_REJECTION is exported."
if it's not acceptable please let me know.

Reply:
@Morteza-Jenabzadeh these errors are pretty useless if we export them since none of the methods use them. And I think we already export the error codes. So let's remove these.

Reply:
I have removed them.